.github/workflows/get-version.yml

```diff
@@ -101,4 +105,6 @@ jobs:
               ;;
           esac
           echo "stability=$STABILITY" >> $GITHUB_OUTPUT
+          echo "env=$MAJOR-$STABILITY" >> $GITHUB_OUTPUT
+          echo "GH_ENV: $MAJOR-$STABILITY"
```

Comment:
Is it still required ?

Reply:
Line 108 is the mandatory output
Line 109 is just a log to known the env name for the workflow. I can remove it if we don't want this kind of log in the workflow.